### PR TITLE
Rename the workspaceone.workspaceone_token to just workspaceone.token

### PR DESCRIPTION
**Pull request by isimluk**
I am not really fixated on this change. Just offering this change for consideration. 

@kylesmartin, feel free to reject really.

### CODE REVIEW

File: config/config.ini [hunk 1]
```diff
@@ -59,7 +59,7 @@
 # Workspace One section is applicable only when Workspace One backend is enabled in the [main] section.
 
 # Uncomment to provide Workspace One token. Alternatively, use WORKSPACEONE_TOKEN variable
```
Comment (shawndwells) [inline config/config.ini:61]:
```suggestion
# Uncomment to provide Workspace One token. Alternatively, use `token` variable
```

Comment (shawndwells) [reply config/config.ini:61]: Comment should be updated if the variable name is changed

File: fig/backends/workspaceone/README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ backends=WORKSPACEONE
 # Workspace One section is applicable only when Workspace One backend is enabled in the [main] section.
 
 # Uncomment to provide Workspace One token. Alternatively, use WORKSPACEONE_TOKEN variable
```
Comment (shawndwells) [inline fig/backends/workspaceone/README.md:16]:
```suggestion
# Uncomment to provide Workspace One token. Alternatively, use `token` variable
```

**Review by shawndwells — COMMENTED** [review]
Small nit to update the comments to reflect new variable name.

File: config/config.ini (hunk 1 above)
Comment (isimluk) [reply config/config.ini:61]: Cannot change the documentation here without changing the code. The environment variable WORKSPACE_TOKEN is still required by code.

Let me step back a bit and describe my thought process about naming things. I prefer to have the environment variables names to be fully qualified. So it is very clear what the environment variable configures. On the other hand our config file is structured, modularized and thus the variable in given config modules does not need to be fully qualified as it is clear that it relates to the given section/module.

**Review by kylesmartin — APPROVED** [review]
I like the change, once its in the workspace one namespace it should just be called "token".

**Pull request merged**